src/main/kotlin/oncall/controller/Controller.kt

```diff
@@ -0,0 +1,47 @@
+package oncall.controller
+
+import oncall.domain.calendar.Calendar
+import oncall.domain.calendar.CustomDayOfWeek
+import oncall.domain.member.Members
+import oncall.domain.service.Scheduler
+import oncall.view.Input
+import oncall.view.Output
+
+class Controller {
+
+    fun handle() {
+        val calendar = retryCallback { createCalendar() }
+        val weekdayOnCallMembers = retryCallback { createWeekdayMember() }
+        val holidayOnCallMembers = retryCallback { createHolidayMember() }
+        val scheduler = Scheduler(calendar, weekdayOnCallMembers, holidayOnCallMembers)
+        val scheduledOrder = scheduler.schedule()
+        Output.printMemberOrder(scheduledOrder, calendar.month)
+    }
+
+    private fun createCalendar(): Calendar {
+        val dayOfWeekAndMonth = Input.enterStartDate()
```

Comment:
```suggestion
        val (dayOfWeek, month) = Input.enterStartDate()
```
This is possible as well